src/core/nis1/external/nacl-fast.js

```diff
@@ -1,4 +1,4 @@
-(function(nacl) {
+(function (nacl) {
```

Comment:
This is a prettier automatic style:fix 
PRs should run 'npm run style:fix' before pushing

Reply:
Update, this is copied code. Let me ignre from prettier.